conf/pacific/upgrades/5_0_upgrade_small-cluster.yaml

```diff
@@ -0,0 +1,42 @@
+globals:
+  - ceph-cluster:
+     name: ceph
+     node1:
+       role:
+         - _admin
+         - mon
+         - mgr
+         - osd
+         - installer
+       no-of-volumes: 1
+       disk-size: 40  
+     node2:
+       role:
+         - mon
+         - mgr
+         - osd
+         - grafana
+       no-of-volumes: 1
+       disk-size: 40
+     node3:
+       role:
+         - mon
+         - mgr
+         - osd
+         - mds
+       no-of-volumes: 1
+       disk-size: 40
+     node4:
+       role:
+         - rgw
+         - iscsi-gw
+
+     node5:
+       role:
+         - rgw
+         - iscsi-gw
+         - client
+
+     node6:
+       role:
+         - nfs
```

Comment:
Shouldn't we remove this node as it is not supported in 5.0?

Reply:
@psathyan Sorry for the mishap
Please check now